linkerd/tracing/src/lib.rs

```diff
@@ -74,26 +60,28 @@ pub fn init_log_compat() -> Result<(), Error> {
 // === impl Settings ===
 
 impl Settings {
-    pub fn from_env() -> Option<Self> {
-        let filter = std::env::var(ENV_LOG_LEVEL).ok();
-        if let Some(level) = filter.as_ref() {
-            if level.to_uppercase().trim() == "OFF" {
-                return None;
-            }
-        }
-
-        Some(Self {
-            filter,
+    pub fn from_env() -> Self {
+        Self {
+            filter: std::env::var(ENV_LOG_LEVEL).ok(),
```

Comment:
IIUC this is only called in one place. Should it just take an `Instant`?

Reply:
good call, changed it to just pass in the `Instant` there!